src/bootstrap/tool.rs

```diff
@@ -343,7 +350,7 @@ tool!(
     Compiletest, "src/tools/compiletest", "compiletest", Mode::ToolTest;
     BuildManifest, "src/tools/build-manifest", "build-manifest", Mode::ToolStd;
     RemoteTestClient, "src/tools/remote-test-client", "remote-test-client", Mode::ToolStd;
-    RustInstaller, "src/tools/rust-installer", "fabricate", Mode::ToolStd;
+    RustInstaller, "src/tools/rust-installer", "fabricate", Mode::ToolStd, llvm_tools = false;
```

Comment:
What makes RustInstaller special? Can we add a comment?

Reply:
@Mark-Simulacrum Added a FIXME in ab5e3e6. The main reason is I've only verified RustInstaller can run without the LLVM binaries in $PATH (as the main purpose is to remove unnecessary work from distcheck).